hydroflow_cli_integration/src/lib.rs

```diff
@@ -28,6 +28,7 @@ use tokio::net::{UnixListener, UnixStream};
 use tokio_util::codec::{FramedRead, FramedWrite, LengthDelimitedCodec};
 
 #[cfg(not(unix))]
+#[allow(dead_code)]
```

Comment:
Hmm in this case do we even need this type?

Reply:
Noo we don't

Reply:
But leaving it there just to keep things consistent